mmseg/core/optimizers/layer_decay_optimizer_constructor.py

```diff
@@ -115,6 +139,9 @@ def add_params(self, params, module):
                 layer_id = get_num_layer_layer_wise(
                     name, self.paramwise_cfg.get('num_layers'))
                 logger.info(f'set param {name} as id {layer_id}')
+            elif decay_type == 'layer_wise_vit':
+                layer_id = get_num_layer_for_vit(name, num_layers)
+                logger.info(f'set param {name} as id {layer_id}')
```

Comment:
I think it might be better to call get num layer function based on transforms name, like
```suggestion
                if `ConvNext` in module.__class__.__name__:
                    layer_id = get_num_layer_for_convnext(
                        name, self.paramwise_cfg.get('num_layers'))
                    logger.info(f'set param {name} as id {layer_id}')
              elif `BEiT` in module.__class__.__name__:
                    layer_id = get_num_layer_for_vit(name, num_layers)
                    logger.info(f'set param {name} as id {layer_id}')
```

Reply:
In future work, perhaps not only BEiT and ConvNeXt could use this optimizer? In addition, ConvNeXt has two ways to get the layer_id. After modification, the corresponding model name in the unit test also needs to be changed, such as BEiTExampleModel -->BEiT. Do we implement these changes?